src/modules/imageresizer/ui/App.xaml.cs

```diff
@@ -13,8 +13,12 @@
 
 namespace ImageResizer
 {
+#pragma warning disable CA1001 // Types that own disposable fields should be disposable
```

Comment:
@enricogior  @crutkas @bricelam ThemeManager is currently disposable, apparently this class is not. I disabled the warning for now, but needs to be fixed - it's just something that goes beyond my skillset :).